legal-api/src/legal_api/resources/v2/business/business.py

```diff
@@ -167,54 +166,26 @@ def search_businesses():
     try:
         json_input = request.get_json()
         identifiers = json_input.get('identifiers', None)
+        search_filter_name = json_input.get('name', None)
+        search_filter_type = json_input.get('type', None)
+        search_filter_status = json_input.get('state', None)
```

Comment:
What about partial match for Number? Eg,

![image](https://github.com/user-attachments/assets/d039210b-c1bc-4e12-963f-818d41be05d3)

Reply:
`json_input.get('identifier_search', None)`